.github/workflows/antora-build.yml

```diff
@@ -0,0 +1,30 @@
+name: Antora Build
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.1
+      with:
+        ref: main
+        fetch-depth: '0'
+    - name: Install Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+    - name: Install Antora
+      run: npm i -g @antora/cli@2.3 @antora/site-generator-default@2.3
```

Comment:
In order to have the full-text search working, in addition to the JS/CSS in our Antora UI, you will need the antora-site-generator-lunr instead of the default site generator, i.e.: `npm i -g @antora/cli@2.3 antora-site-generator-lunr`